ulog/README.md

```diff
@@ -13,11 +18,14 @@ import "go.uber.org/fx/ulog"
 
 func main() {
   // Initialize logger object
-  log := ulog.Logger()
+  logBuilder := ulog.Builder()
 
   // Optional, configure logger with configuration preferred by your service
   logConfig := ulog.Configuration{}
-  log.Configure(&logConfig)
+  logBuilder := logBuilder.WithConfiguration(&logConfig)
```

Comment:
Oh. you are right. I'll modify the snippet 
`  // Configure logger with configuration preferred by your service
  logConfig := ulog.Configuration{}
  logBuilder := ulog.Builder().WithConfiguration(&logConfig)
`